src/proto/message.rs

```diff
@@ -98,12 +119,23 @@ impl Encode for SignRequest {
     }
 }
 
+/// A container for a public / private key pair, or a certificate / private key.
+///
+/// When adding an identity to an agent, a user can provide either:
+/// 1. A public / private key pair
+/// 2. An OpenSSH [certificate](https://github.com/openssh/openssh-portable/blob/master/PROTOCOL.certkeys)
+///
+/// This structure covers both types of identites a user may
```

Comment:
```suggestion
/// This structure covers both types of identities a user may
```

I'm quite confused why the "Check spelling" CI job didn't catch it since I did running it locally :roll_eyes:

Reply:
good pickup!